client/layout/index.js

```diff
@@ -60,30 +73,38 @@ class Layout extends Component {
 	}
 
 	render() {
-		const { isEmbeded, ...restProps } = this.props;
+		const { isEmbedded, ...restProps } = this.props;
 		return (
 			<div className="woocommerce-layout">
 				<Slot name="header" />
 				<TransientNotices />
-
-				<div className="woocommerce-layout__primary" id="woocommerce-layout__primary">
-					{ window.wcAdminFeatures[ 'store-alerts' ] && <StoreAlerts /> }
-					<Notices />
-					{ ! isEmbeded && (
+				{ ! isEmbedded && (
+					<PrimaryLayout>
 						<div className="woocommerce-layout__main">
 							<Controller { ...restProps } />
 						</div>
-					) }
-				</div>
+					</PrimaryLayout>
+				) }
 			</div>
 		);
 	}
 }
 
 Layout.propTypes = {
-	isEmbededLayout: PropTypes.bool,
+	isEmbedded: PropTypes.bool,
 };
 
+export class NoticeArea extends Component {
```

Comment:
I this component used somewhere? I couldn't find any file importing it.

Reply:
Nice catch - I forgot to remove that.